packages/pro-components/chat/chat-action.tsx

```diff
@@ -24,22 +24,23 @@ export default defineComponent({
     const renderTNodeJSX = useTNodeJSX();
     const messagePluginInstance = MessagePluginSingleton.getInstance();
     const { globalConfig } = useConfig('chat');
-    const { copyTipText, likeTipText, dislikeTipText, refreshTipText, copyTextSuccess, copyTextFail } = toRefs(
-      globalConfig.value,
-    );
+
     return () => {
       // textLoading更新后要传给子组件和孙组件
       const content = renderTNodeJSX('content');
+      const { copyTipText, likeTipText, dislikeTipText, refreshTipText, copyTextSuccess, copyTextFail } =
+        globalConfig.value;
+
       // 内置操作按钮，assistantActions和插槽判断 t-chat注入的属性获取不到默认为false
       const disabled = props.disabled;
       const copyAnswer = () => {
         // 根据e获取当前按钮选择器
         const copyBtn = new Clipboard(`.copy-btn`);
         copyBtn.on('success', () => {
-          messagePluginInstance.showSuccess(copyTextSuccess.value);
+          messagePluginInstance.showSuccess(copyTextSuccess);
```

Comment:
```suggestion
          messagePluginInstance.showSuccess(globalConfig.value.copyTextSuccess);
```
也成globalConfig.value.xx吗

Reply:
return 里结构没关系 可以保留响应式

Reply:
用法统一点